openadapt/window/_windows.py

```diff
@@ -99,16 +97,16 @@ def get_active_window(depth=10, max_width=10, filename=None) -> Desktop:
     return window
 
 
-def get_element_properties(element):
-    """
-    Recursively retrieves the properties of each element and its children.
+def get_element_properties(element: Any) -> dict:
```

Comment:
also should be pywinauto.application.Application

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/6159a69dd7340dd520bd020a57101dfd9d444c9f